src/components/Avatar.js

```diff
@@ -5,5 +5,11 @@
 export default function Avatar(props) {    
     const { person } = props;
 
-    // return ...
+    return (
+        <img
+            classname = "Avatar"
```

Comment:
- There shouldn't be any space before and after equal sign
- wrong alt attribute value: We use alt attribute in img element to make them accessible, they should have a meaningful value which represents the image